syntax = "proto3";

package execution;

message PoolExecutorConfig {
    uint32 num_workers = 1;
}

message AffinityExecutorConfig {
    uint32 num_lanes = 1;
}

message Config {
    PoolExecutorConfig pool_executor = 1;
    AffinityExecutorConfig affinity_executor = 2;
}
